T/Torch/build_tarballs.jl

```diff
@@ -0,0 +1,55 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Torch"
+version = v"1.4.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/dhairyagandhi96/Torch.jl.git", "985c4280c17fd9b0af978760be66761456a0113b"),
+    ArchiveSource("https://download.pytorch.org/libtorch/cu101/libtorch-cxx11-abi-shared-with-deps-1.4.0.zip", "f214bfde532877aa5d4e0803e51a28fa8edd97b6a44b6615f75a70352b6b542e"),
+    ArchiveSource("https://github.com/JuliaGPU/CUDABuilder/releases/download/v0.3.0/CUDNN+CUDA10.1.v7.6.5.x86_64-linux-gnu.tar.gz", "79de5b5085a33bc144b87028e998a1d295a15c3424d6d45b25defe500f616974", unpack_target = "cudnn"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+mv libtorch/ ${prefix}
+cd /usr/local/
+mkdir cuda
+cd cuda
+mkdir lib64
+cd lib64/
+ln -s ${prefix}/lib64/libcudart.so libcudart.so
+ln -s ${prefix}/lib64/libnvToolsExt.so libnvToolsExt.so
+cd $WORKSPACE/srcdir
+mv cudnn $prefix
+cd $WORKSPACE/srcdir/Torch.jl/build
+mkdir build && cd build
+cmake -DCMAKE_PREFIX_PATH=$prefix -DTorch_DIR=$prefix/libtorch/share/cmake/Torch -DCUDA_TOOLKIT_ROOT_DIR=$prefix -DCUDNN_LIBRARY_PATH=$prefix/cudnn/lib/libcudnn.so  -DCUDNN_INCLUDE_DIR=$prefix/cudnn/include ..
+cmake --build .
+mkdir -p "${libdir}"
+cp -r $WORKSPACE/srcdir/Torch.jl/build/build/* "${libdir}"
```

Comment:
Related to [this comment](https://github.com/JuliaPackaging/Yggdrasil/pull/770#issuecomment-611292061), I assume that here we can just do
```suggestion
cp -r $WORKSPACE/srcdir/Torch.jl/build/build/*.${dlext} "${libdir}"
```
no?